### DISCUSSION TITLE
(Someone delete this pls)

### DISCUSSION THREAD

**Discussion by Nonethousand** [ScratchAddons/ScratchAddons#5119] [Show and tell]
can't figure out how to close this, so just don't post here

**Comment by BroJac5246** [comment]
This really isn't the place to ask, this is for Scratch Addons.

**Reply by Nonethousand** [reply]
I searched for posts like this and found some, so I thought it was ok.

**Reply by ghost** [reply]
Scratch forums under advanced topics is a great place to go. That community has a bunch of CRAZY good developers willing to help anyone out.
